lib/kill.js

```diff
@@ -18,9 +18,13 @@ const setKillTimeout = (kill, signal, options, killResult) => {
 	}
 
 	const timeout = getForceKillAfterTimeout(options);
-	setTimeout(() => {
+	const t = setTimeout(() => {
 		kill('SIGKILL');
-	}, timeout).unref();
+	}, timeout);
+
+	if (t.unref) {
```

Comment:
```suggestion
// istanbul ignore else
if (t.unref) {
```

We don't run tests on Electron, so this branch cannot be covered by Ava/Istanbul.